clang/lib/Format/FormatToken.cpp

```diff
@@ -71,8 +71,21 @@ bool FormatToken::isSimpleTypeSpecifier() const {
   }
 }
 
-bool FormatToken::isTypeOrIdentifier() const {
-  return isSimpleTypeSpecifier() || Tok.isOneOf(tok::kw_auto, tok::identifier);
+// Sorted common C++ non-keyword types.
+static SmallVector<StringRef> CppNonKeywordTypes = {
+    "int16_t",  "int32_t",  "int64_t",  "int8_t",  "intptr_t",  "size_t",
```

Comment:
ssize_t too? should everything be considered? https://en.cppreference.com/w/cpp/types/integer

Reply:
`ssize_t` is a POSIX type. Regarding the fixed-width integer types, I wanted to start with a few common ones. (I will add `ptrdiff_t` though.)